wdl/MELT.wdl

```diff
@@ -600,12 +600,16 @@ task RunMELT {
     set -Eeuo pipefail
 
     # MELT expects the BAM index to have extension ".bam.bai"
-    mv ~{bam_file} ~{sample_id}.bam
-    mv ~{bam_index_file} ~{sample_id}.bam.bai
+    # Converting to cp for FSx/Local
+    cp ~{bam_file} ~{sample_id}.bam
+    cp ~{bam_index_file} ~{sample_id}.bam.bai
 
     # these locations should be stable
     MELT_DIR="/MELT"
-    CROMWELL_ROOT="/cromwell_root"
+    # Adding sample id to ensure the MELT files are created in a separate folder even if multiple jobs run on same compute instance.
+    # Hence adding mkdir to create the folder
+    CROMWELL_ROOT="/cromwell_root/~{sample_id}"
```

Comment:
It is on our to-do list to clean up the absolute paths in this workflow. We may request further changes in relation to this, or we may make those changes ourselves later on

Reply:
@epiercehoffman : are any changes needed in this from our side for your testing? It shouldn't be an issue on your end as well.

Reply:
Not worth fixing since we are deprecating melt support in the near feature. Your changes here are ok.